src/utils/store.js

```diff
@@ -44,13 +44,13 @@ function get(key) {
     if (!isLocalStorageSupported) return
     try {
       var value = window.localStorage[`${NAMESPACE}.${key}`]
+
+      if (value) {
+        return JSON.parse(value)
```

Comment:
Could/should it be `_JSON` like in the [set](https://github.com/missive/emoji-mart/blob/master/src/utils/store.js#L35) function?